### DISCUSSION TITLE
Blazor Bootstrap 2.1.0 Example Modal with form

### DISCUSSION THREAD

**Discussion by leoprogramacion2021** [vikramlearning/blazorbootstrap#626] [Q&A]
I'm new to this, I'd like to see an example of how to make a data registration form in a Modal, and then consume the API service, with Blazor Boostrap 2.1.0, thanks

**Comment by gvreddy04** [comment]
@leoprogramacion2021 Thanks for trying **BlazorBootstrap**. I'll share an example ASAP."

**Comment by leoprogramacion2021** [comment]
The solution was tried, it's perfect!
